internal/kafka/internal/services/clusters.go

```diff
@@ -763,15 +764,15 @@ type ClusterSelection struct {
 	SupportedInstanceType string
 	DynamicCapacityInfo   api.JSON
 	Status                string
+	ClusterType           string
```

Comment:
Side comment: What is the ClusterType when a cluster is not enterprise?

Reply:
`managed`

Reply:
Is a manual cluster always managed?

Reply:
right now we only have two types of clusters regardless of scaling policy (enterprise or non-enterprise (managed))

Reply:
Yes. Clusters registered via the manual list are managed clusters i.e our own clusters and not user supplied clusters.